public-ui/src/app/data-browser/views/error-message/error-message-react.component.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { ClrIcon } from 'app/utils/clr-icon';
+import * as React from 'react';
+
+interface Props {
+    dataType: string;
+}
+
+export class ErrorMessageReactComponent extends React.Component<Props, {}> {
+
+  constructor(props: Props) {
+    super(props);
+  }
+
+  render() {
+    const {dataType} = this.props;
+    const iconClass = 'alert-icon';
+    const iconShape = 'exclamation-triangle';
```

Comment:
Nit: Since these are only used once, just put them directly on the element.

Reply:
Changed this.

Reply:
Can we remove these since they're now unused?